src/fprime/util/build_helper.py

```diff
@@ -71,7 +71,7 @@ def validate(parsed, unknown):
         }
         cmake_args.update(d_args)
     # Build type only for generate, jobs only for non-generate
-    elif parsed.command not in ["info", "purge", "hash-to-file", "new"]:
+    elif parsed.command not in ["info", "purge", "hash-to-file", "new", "new-port"]:
```

Comment:
It looks like the you use `new` for new ports a well. Is `new-port` used?

Reply:
That must have been left over from my initial new port template, but I'll remove it now